### PR TITLE
wget: add Hindi translation

### PR DESCRIPTION
**Pull request by adityarjadhav**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.hi/common/wget.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+`wget --user={{username}} --password={{password}} {{https://example.com}}`
+
+- अधूरा डाउनलोड जारी रखें:
+
+`wget --continue {{https://example.com}}`
+
+- टेक्स्ट फ़ाइल में संग्रहीत सभी URL को एक विशिष्ट निर्देशिका में डाउनलोड करें:
+
+`wget --directory-prefix {{path/to/directory}} --input-file {{URLs.txt}}`
```
Comment (kbdharun) [inline pages.hi/common/wget.md:37]:
```suggestion
`wget --directory-prefix {{निर्देशिका/का/पथ}} --input-file {{URLs.txt}}`
```

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution. The translations LGTM, Just a small suggestion.

File: pages.hi/common/wget.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+`wget --mirror --no-parent {{https://example.com/somepath/}}`
+
+- डाउनलोड की गति और कनेक्शन के पुन: प्रयास की संख्या सीमित करें:
+
+`wget --limit-rate={{300k}} --tries={{100}} {{https://example.com/somepath/}}`
+
+- मूल प्रमाणीकरण का उपयोग करके किसी HTTP सर्वर से फ़ाइल डाउनलोड करें (FTP के लिए भी काम करता है):
+
+`wget --user={{username}} --password={{password}} {{https://example.com}}`
```
Comment (navarroaxel) [inline pages.hi/common/wget.md:29]:
```suggestion
`wget --user={{उपयोगकर्ता_नाम}} --password={{password}} {{https://example.com}}`
```

Comment (kbdharun) [reply pages.hi/common/wget.md:29]: @adityarjadhav Can you translate `password` placeholder too?

**Review by kbdharun — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**